### DISCUSSION TITLE
visualization renders locally but not on github pages

### DISCUSSION THREAD

**Discussion by yzse** [community/community#58334] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

hi, would someone mind telling why my networks visualization is not rendering on gh-pages?  I'm able to get other apps (the table) to load, but not the visualization.

**https://github.com/yzse/global-nuclear-trade**

I've tripled-checked all the routes & directories and cannot find anything wrong.  I've even took out at the .css and .js files out of their folder and put them on the same directory with index.html, but it does not seem to work.

I've also added the following metadata in the headers with no luck:
`<meta http-equiv=“X-UA-Compatible” content=“ie=edge”>`
`<meta name=“viewport” content=“width=device-width, initial-scale=1.0”>`

my repo is modelled after [this one](https://github.com/harmslab/NetworkViewer/blob/master/index.html), if it helps.

**Comment by airtower-luna** [comment, marked as answer]
Some part of your code tries to load `https://yzse.github.io/global-nuclear-trade/templates/modal.html`, and there's no `templates/modal.html` in your repository, so that fails, and causes and exception in the script. I assume you'll have to add the missing file, but I didn't dig deeper. :wink: 

As for those headers, I'm not sure if they're useful at all for what you're doing, but if you want them to work you definitely need *plain* quotes (`""`) around the attributes, not curved quotes (`“”`). Otherwise they simply add syntax errors. :sweat_smile:

**Reply by yzse** [reply]
Wow, that worked, thanks!  I removed `templates/modal.html` since I wasn't using it, guess I missed it somewhere.

Just for future reference - how were you able to figure this out and find exceptions in the script?  This was working perfectly in my local machine so I wasn't sure how to troubleshoot it as I'm deploying it.

**Reply by airtower-luna** [reply]
I used the developer tools in the browser ([Firefox in my case](https://firefox-source-docs.mozilla.org/devtools-user/index.html)). The console shows errors, and the network monitor which requests work or fail (and some other things). And for the request that failed with 404 and cause the exception I just checked the path against the repository.

And naturally I was looking at the deployed page. If that doesn't work but your local copy does, you'll want to find differences.
